tensorflow/python/keras/benchmarks/keras_examples_benchmarks/README.md

```diff
@@ -0,0 +1,192 @@
+# Benchmarks for keras model examples
+
+- [Benchmarks for keras model examples](#benchmarks-for-keras-model-examples)
+  - [Keras benchmarks](#keras-benchmarks)
+    - [Available models](#available-models)
+      - [Computer Vision examples](#computer-vision-examples)
+      - [Text & Sequence examples](#text--sequence-examples)
+      - [Other examples](#other-examples)
+    - [Available benchmark results](#available-benchmark-results)
+      - [Cifar10 CNN benchmark](#cifar10-cnn-benchmark)
+      - [MNIST Conv benchmark](#mnist-conv-benchmark)
+      - [MNIST Hierarchical RNN (HRNN) benchmark](#mnist-hierarchical-rnn-hrnn-benchmark)
+      - [Bidirectional LSTM benchmark](#bidirectional-lstm-benchmark)
+      - [Text classification with transformer benchmark](#text-classification-with-transformer-benchmark)
+      - [MLP benchmark](#mlp-benchmark)
+      - [Antirectifier benchmark](#antirectifier-benchmark)
+      - [IRNN benchmark](#irnn-benchmark)
+  - [Install Bazel](#install-bazel)
+  - [Run benchmarks](#run-benchmarks)
+  - [Add new benchmarks](#add-new-benchmarks)
+  - [Troubleshooting](#troubleshooting)
```

Comment:
Is it better not having the TOC, to make the README shorter?

Reply:
But if there have too many sections, we should give the TOC and the user will check the TOC first and easy to guide.

Reply:
Okay, sgtm.